boards/arm/sama5/sama5d2-xult/Kconfig

```diff
@@ -25,8 +25,7 @@ endchoice # CPU Frequency
 
 config SAMA5D2XULT_USBHOST_STACKSIZE
 	int "USB host waiter stack size"
-	default 1536 if USBHOST_HUB
-	default 1024 if !USBHOST_HUB
```

Comment:
It is inappropriate to increase the size of this tuned stack size.  Let's not do that.